packages/core/src/components/table/table.css

```diff
@@ -0,0 +1,109 @@
+.hds-table-container {
+  overflow-x: auto;
+}
+
+.hds-table {
+  border-collapse: collapse;
+  font-family: var(--font-default); /* Tartteeko laittaa? */
+  width: 100%;
+  line-height: 1.5;
+}
+
+.hds-table--dark {
+  --background-color: var(--color-bus);
+}
+
+.hds-table--light {
+  --background-color: var(--color-silver);
+}
+
+.hds-table th {
+  border-bottom: 1px solid var(--color-black-20);
+  white-space: nowrap;
+  text-align: left;
+  font-weight: 500;
+  font-size: 16px;
+  padding: 12px 24px;
+}
+
+.hds-table.hds-table--dark th {
+  color: var(--color-white);
+}
+
+.hds-table.hds-table--light th {
+  color: var(--color-black-90);
+}
+
+.hds-table--zebra .hds-table__content tr:nth-child(even) td {
+  background-color: #f2f2f2;
+}
+
+.hds-table--with-vertical-lines .hds-table__content td {
+  border-right: 1px solid var(--color-black-50);
+}
+
+.hds-table--with-vertical-lines .hds-table__content {
+  border-left: 1px solid var(--color-black-50);
+}
+
+.hds-table--with-vertical-lines.hds-table--with-vertical-header .hds-table__content {
+  border-left: none;
+}
+
+.hds-table--with-vertical-lines .hds-table__header-row {
+  border-right: 1px solid var(--color-bus);
+  border-left: 1px solid var(--color-bus);
+}
+
+.hds-table--with-vertical-lines.hds-table--with-vertical-header .hds-table__header-row {
+  border-left: none;
+}
+
+.hds-table--with-vertical-lines.hds-table--light .hds-table__header-row {
```

Comment:
A suggestion: Would it make this selector more readable if it would start with the class that makes it different? In this case, it would be .hds-table--light.

Reply:
- [ ] Start with the class that makes it different from previous